Sources/Hedera/Execute.swift

```diff
@@ -119,6 +122,13 @@ internal func executeAny<E: Execute & ValidateChecksums>(_ client: Client, _ exe
         maxElapsedTime: .limited(timeout)
     )
 
+    var firstTransactionId: TransactionId? = nil
+
+    // Keep track of the initialTransactionId to reference in chunked Transactions.
+    if let initialTransactionId = initialTransactionId {
+        firstTransactionId = initialTransactionId
+    }
+
```

Comment:
```suggestion
```
instead pass this directly to `executeAnyInner` (except don't do that either, see my above comment)